feat(docs): update pwa guide and other metro resources (#26475)
expo/expo: docs/pages/guides/minify.mdx

The default minification of Expo CLI is sufficient for most projects. However, you can customize the minifier to optimize for speed or remove additional features like logs.

## Remove console logs

> You may want to remove console logs from your production build. This can be done by using the `drop_console` option in the Terser minifier config.

Reviewer: ```suggestion
You can remove console logs from your production build. Use the `drop_console` option in the Terser minifier config.
```
